source/speech/__init__.py

```diff
@@ -1061,6 +1061,10 @@ def speakTextInfo(
 		onlyInitialFields,
 		suppressBlanks
 	)
+
+	if reason == controlTypes.REASON_SAYALL:

```

Comment:
As we discussed, if this is only being kept for backwards compat, please leave a comment that it is deprecated and will be removed in 2021.1. Please add label `deprecated/2021.1` to this PR so it can tracked.

For anyone interested in this process see https://github.com/nvaccess/nvda/wiki/deprecations